config/jupyterhub_config.py

```diff
@@ -54,40 +59,58 @@
     CustomDockerSpawner.RW_MINIO_GROUP: [],
 }
 
-c.JupyterHub.spawner_class = CustomDockerSpawner
-
-c.DockerSpawner.hub_connect_url = f"http://{os.environ['SPARK_DRIVER_HOST']}:{os.environ['NOTEBOOK_PORT']}"
-# Set the Docker image to use for user containers
-c.DockerSpawner.image = os.environ['JUPYTERHUB_USER_IMAGE']
-
-c.DockerSpawner.cmd = ['echo', 'Starting JupiterHub Single User Server With DockerSpawner ...']
-
-# Container resource limits
-c.DockerSpawner.cpu_limit = 4
-c.DockerSpawner.mem_limit = '16G'
-
-c.DockerSpawner.http_timeout = 120  # 2 minutes (default is 30 seconds)
-c.DockerSpawner.start_timeout = 300  # 5 minutes (default is 60 seconds)
-
-# The network name that Docker containers will use to communicate
-network_name = os.environ.get('NETWORK_NAME')
-if network_name:
-    c.DockerSpawner.network_name = network_name
-c.DockerSpawner.use_internal_ip = True
 environment = os.environ.get('ENVIRONMENT', 'prod').lower()
-# for troubleshooting purposes, keep the container in non-prod environment
-c.DockerSpawner.remove = environment != 'dev'
-c.DockerSpawner.debug = True
-
-# Set extra labels in order to use Rancher's network policies
-# ref: https://rancher.com/docs/rancher/v1.6/en/rancher-services/networking/#containers-created-with-the-docker-cli
-c.DockerSpawner.extra_create_kwargs = {
-    'labels': {
-        'io.rancher.container.network': 'true'
-    }
-}
 
-c.DockerSpawner.shutdown_no_activity_timeout = 60 * 60  # 1 hour
```

Comment:
This line seems to be completely gone, is that what you want?

Reply:
yea. No longer needed. It was controlled by spawner for both docker and kube.

Reply:
:+1: